### PR TITLE
Finished Student Dashboard UI

### PR DESCRIPTION
**Pull request by shomzzz**
- added custom colors: plm colors based on figma design
- worked on student ui dashboard
- new layout and template for navbar and sidebar made by Martinez

### CODE REVIEW

**Review by jhaimec — COMMENTED** [review]
Paayos muna ng conflicts @james-zenitram @shomzzz

**Comment by shomzzz** [comment]
okss na yung conflicts, padouble check nalangg. tnx

File: .env.example
Comment (jhaimec) [inline .env.example:1]: Wag tanggalin to

Comment (jhaimec) [reply .env.example:1]: @shomzzz palagay ulit to, need to

Comment (jhaimec) [reply .env.example:1]: ako na nag add

**Pull request merged**